docs/components/movement-sensor/adxl345.md

```diff
@@ -143,13 +142,13 @@ Then remove and fill in the attributes as applicable to your movement sensor, ac
 ## Attributes
 
 <!-- prettier-ignore -->
-| Name                  | Type   | Inclusion    | Description |
-| --------------------- | ------ | ------------ | ----------- |
-| `board`               | string | **Required** | The `name` of the [board](/components/board/) to which the device is wired. |
-| `i2c_bus`             | string | **Required** | The `name` of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on the [board](/components/board/) wired to this device. |
-| `use_alternate_i2c_address` | bool   | Optional     | Depends on whether you wire SDO low (leaving the default address of 0x53) or high (making the address 0x1D). If high, set true. If low, set false or omit the attribute. <br> Default: `false` |
-| `tap`                 | object | Optional     | Holds the configuration values necessary to use the tap detection interrupt on the ADXL345. See [Tap attributes](#tap-attributes). |
-| `free_fall`           | object | Optional     | Holds the configuration values necessary to use the free-fall detection interrupt on the ADXL345. See [Freefall attributes](#freefall-attributes). |
+| Name | Type   | Inclusion    | Description |
+| ---- | ------ | ------------ | ----------- |
+| `i2c_bus` | string | **Required** | The `name` of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on the [board](/components/board/) wired to this device. |
+| `board` | string | Optional | The `name` of the [board](/components/board/) to which the device is wired. Only needed if you've configured any [interrupt](/components/board/#digital_interrupts) functionality. |
+| `use_alternate_i2c_address` | bool | Optional | Depends on whether you wire SDO low (leaving the default address of 0x53) or high (making the address 0x1D). If high, set true. If low, set false or omit the attribute. <br> Default: `false` |
```

Comment:
I'm open to disagreement, but I'd swap lines 148 and 149. That way, the board is right next to the tap and free_fall sections that interact with it, and the fields are roughly ordered by popularity (use_alternative_i2c_address is probably used more often than board will be after this change is out).

Reply:
Done!